SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -658,20 +657,29 @@ void USpatialNetDriver::OnLevelAddedToWorld(ULevel* LoadedLevel, UWorld* OwningW
 		return;
 	}
 
-	// If we have authority over the GSM when loading a sublevel, make sure we have authority
-	// over the actors in the sublevel.
-	if (GlobalStateManager != nullptr)
+	check(GlobalStateManager != nullptr)
```

Comment:
I thought servers loaded levels before connecting to Spatial. In which case, this would be null.

Reply:
We removed that, since internally it checked for authority on the SINGLETON_MANAGER_COMPONENT, which is a bit misleading if you want to check authority on other components that the GSM manages.